README.md

```diff
@@ -30,8 +30,8 @@ Mongoose storage driver extention for testing of **S3 type storages**. The repo
 
 * API version: 2006-03-01
 * Authentification:
-    * [v2](https://docs.aws.amazon.com/general/latest/gr/signature-version-2.html) (by default)
-    * [v4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html)
+    * [v2](https://docs.aws.amazon.com/general/latest/gr/signature-version-2.html) 
+    * [v4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) (by default)
```

Comment:
Wouldn't the default version be 2? I assume it based on the following line: https://github.com/emc-mongoose/mongoose-storage-driver-s3/pull/12/files#diff-20822718f67944a02ff4f979e6ae8cd6d9039ae75eaf41111f2f298bb918699cR179

Reply:
You are right. I reverted it with the last commit, but forgot to change back the doc, thanks